doc/api/child_process.md

```diff
@@ -277,7 +277,7 @@ const child = exec('grep ssh', { signal }, (error) => {
 controller.abort();
 ```
 
-### `child_process.execFile(file[, args][, options][, callback])`
+### `child_process.execFile(file, [args], options, callback])`
```

Comment:
This moves the brackets to the wrong place. `foo(bar[, baz])` means `foo(bar)` or `foo(bar, baz)`. If you change it to `foo(bar, [baz])` then `foo(bar, )` is indicated to be valid.